### PR TITLE
Scaffold archive exhibition

### PR DESCRIPTION
**Pull request by mikeyrayvon**
also:
- refactor queries on front-page
- refactor some js from last branch
- add random dot-on-hover to some items (maybe more cute if it follows the cursor?)

### CODE REVIEW

File: archive-exhibition.php [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+?>
+
+<main id="main-content">
+  <section id="posts">
+    <div class="container">
+
+<?php
+$now = time();
+
+$current_args = array(
```
Comment (varffs) [inline archive-exhibition.php]: are these args the same as frontpage logics? Perhaps worth making a function that returns either the WP_Queries or maybe these arg arrays?

File: src/js/main.js [hunk 2]
```diff
@@ -84,19 +86,54 @@ class Site {
...
     }
   }
 
+  positionHoverDots() {
+    var _this = this;
+
+    var offset = _this.dotDiameter + 10;
+
+    _this.$hoverDot.each(function() {
+      var randomTop = _this.randomInt(100 - offset, offset); // min offset% from any edge
```
Comment (varffs) [inline src/js/main.js]: when designing not defined stuff like this I'm very pro putting values like this `100` as set variables saved on the object/class

File: src/styl/responsive/mobile.styl [hunk 3]
```diff
@@ -26,7 +39,9 @@ $border-style = 1px solid $color-black
 
 // FRONT PAGE
 
-.frontpage-section-label
+.frontpage-section-label,
+.exhibitions-section-label
```
Comment (varffs) [inline src/styl/responsive/mobile.styl]: maybe these are just `.section-label`s?

Comment (mikeyrayvon) [reply src/styl/responsive/mobile.styl]: look at the whole PR

**Pull request merged**